### PR TITLE
ci(repo): migrate to github actions

### PR DESCRIPTION
**Pull request by bengaal**
Migrate from travis to github actions.

### CODE REVIEW

File: invenio_rdm_pure/version.py [hunk 1]
```diff
@@ -11,4 +11,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "1.0.0.dev20200000"
+__version__ = "0.1.0"
```
Comment (mb-wali) [inline invenio_rdm_pure/version.py:14]: why change the version - but not a big deal.

File: run-tests.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@
 docker-services-cli up postgresql es redis && \
```
Comment (mb-wali) [inline run-tests.sh:9]:
```suggestion

```

Comment (mb-wali) [inline run-tests.sh:9]:
```suggestion
docker-services-cli --verbose up es postgresql redis && \
```
using --verbose mode

File: run-tests.sh [hunk 3]
```diff
@@ -9,7 +9,7 @@
...
 pydocstyle invenio_rdm_pure tests docs && \
```
Comment (mb-wali) [inline run-tests.sh:10]:
```suggestion

```

File: setup.py [hunk 4]
```diff
@@ -109,6 +109,7 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```
Comment (mb-wali) [inline setup.py]:
```suggestion

```
we are not sure if this version is yet compatible.

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8, 3.9]
```
Comment (mb-wali) [inline .github/workflows/tests.yml]:
```suggestion
          python-version: [3.6, 3.7, 3.8]
          requirements-level: [min, pypi]
```

**Review by mb-wali — CHANGES_REQUESTED** [review]
Request changes:
1. revert all the changes from ```README.rst``` file.
**because**- first we have a PR #26  for this - you can add these changes to that, second you should not remove the coverage badge.
2. remove file ``` invenio_rdm_pure/.github/workflows/pypi-publish.yml```
**because**-  this file has instructions to push the package to PyPI. we don't want that yet. also wrong location.
3. remove file ```invenio_rdm_pure/.github/workflows/tests.yml```
**because** wrong location.
Also see the comments. 

- once changes are done - we will see if something is wrong with the tests failing.

File: invenio_rdm_pure/version.py (hunk 1 above)
Comment (bengaal) [reply invenio_rdm_pure/version.py:14]: I just tried to keep it consistent with https://github.com/inveniosoftware/cookiecutter-invenio-module. This was changed [here](https://github.com/inveniosoftware/cookiecutter-invenio-module/commit/e3ca55f18a3e35ed4d3f5875d9647d408c45295b).

File: setup.py [hunk 6]
```diff
@@ -19,6 +19,8 @@
     "invenio-search[elasticsearch7]>=1.4.0",
     "invenio-config>=1.0.3",
     "SQLAlchemy-Utils>=0.33.1,<0.36",  # FIXME without this: ContextualVersionConflict {invenio-files-rest}
+    "docker-services-cli>=0.2.1,<0.3.0",  # TODO: remove once a new release is out
+    "invenio-assets<1.2.0,>=1.1.2",
```
Comment (mb-wali) [inline setup.py:23]:
```suggestion
    "invenio-assets>=1.2.0",
```

**Review by mb-wali — COMMENTED** [review]
The module is using ```invenio-rdm-records``` of version 0.23.4 - change this to 0.24.1 and remove the ```invenio-assets``` from dependencies.

**Review by mb-wali — APPROVED** [review]

**Pull request merged**